lib/cli.js

```diff
@@ -78,7 +81,7 @@ npmCheck(options)
         currentState.inspectIfDebugMode();
 
         if (options.update) {
-            return update(currentState);
+            return update(currentState, options.installer);
```

Comment:
`currentState` saves all of the `options`, check out `util/state.js` to see how to use it. The `installer` value *must* be in the `defaults` for `set` and `get` to work.